ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,56 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: 3scale-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: 3scale-amp-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      AWS_REGION: us-east-2
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      DEPL_BUCKET_NAME: 3scale-apimanager-s3-bucket
+      DEPL_PROJECT_NAME: threescale
+      OPERATORS: |
+        [
+            {"name": "3scale-operator", "source": "redhat-operators", "channel": "threescale-2.13", "install_namespace": "threescale", "target_namespaces": "threescale"}
+        ]
+      SPOT_INSTANCES: "true"
+    post:
+    - ref: gather-aws-console
+    - ref: 3scale-apimanager-uninstall
+    - chain: ipi-deprovision
+    pre:
+    - chain: ipi-conf-aws
+    - chain: ipi-install
+    - ref: install-operators
```

Comment:
We should decide if we want to consider the operator install and other scenario setup as a test or not mainly for consistency.
From the OCP CI docs:
`The pre chain is intended to be used to set up a testing environment (such as creating a test cluster), the test chain is intended to contain all tests that a job wants to run... If a step in pre or test fails, all pending pre and test steps are skipped and all post steps are run to ensure that resources are properly cleaned up. `

It seems the behavior of pre: and test: will be the same so its really up to a style here I think.

We've put install-operators under the test section in the [MTR scenario](https://github.com/openshift/release/blob/master/ci-operator/config/windup/windup-ui-tests/windup-windup-ui-tests-main__mtr-ocp4.13-lp-interop.yaml#L43) already

I personally like keeping the pre section for just Openshift deployments, then anything else needed specific to testing the scenario, operator installs, crds, tests and stuff like that will all be under test. I think this gives us a good way to quickly view a config file and see the scenario specific steps without being distracted by some being in pre and others in test.

Reply:
I agree with keeping the `install-operators` ref in the `test` stanza rather than in `pre`. Mainly because of the reasons @ascerra has stated, but also because it crowds the config because you aren't just calling the `ipi-aws` workflow, you have to create that workflow in your config file by using the `pre` and `post` stanzas. Not to say that that isn't allowed, but I wouldn't do it if you don't have to.

Reply:
Is there any best practices or doc to follow as to where the keep the operator install ref? or what other teams are doing. I do agree with the reason @ascerra stated

Reply:
Added `install-operator` and `apimanger-install` in test as discussed.  And `apimanager-uninstall` in post